### DISCUSSION TITLE
E-Forms-Notice-Editor --- edit a notice

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#150] [Q&A] [answered]
Hi All !
I installed the demo application, and I saw how html is made from json files.
Just a question :
perhaps I did not see well but it seems that , in this version, it is not possible to edit a notice, is it ?
Thanks a lot
Renato

**Comment by rouschr** [comment, marked as answer]
The editor (demo) currently only shows how to create the form for the creation of a new notice.
This is also just one way to do it and has been done keeping the demo factor in mind (using basic browser features, no extra libs, no Angular, ...).
I would not recommend reusing this as-is for production, but you can certainly get a lot of ideas from it.
In our roadmap we plan the following features next:

1. saving a notice to XML and validating against XSD, see discussions here: https://github.com/OP-TED/eForms-SDK/discussions/126
2. loading an existing notice

Loading a notice will use the xpathAbsolute and xpathRelative of fields.json to fill the form.
So first create the empty form and then fill it. A bit more code is required for the repeating fields.
